src/main/kotlin/racing/CarRacing.kt

```diff
@@ -0,0 +1,29 @@
+package racing
+
+class CarRacing {
+    fun startRacing() {
+        val (carCount, moveCount) = CarRacingInputView().input()
+
+        val cars: List<Car> = createCars(carCount)
+        moveCars(cars = cars, moveCount = moveCount)
+
+        CarRacingResultView().printCarRacingResult(cars = cars, moveCount = moveCount)
+    }
+
+    fun moveCars(cars: List<Car>, moveCount: Int) {
+        for (moveIndex in 0 until moveCount) {
+            cars.forEach { car ->
+                car.go()
+            }
```

Comment:
```suggestion
            cars.forEach(Car::go)
```
메소드 레퍼런스를 적절하게 사용하면 조금 더 간결해 지는 경우가 있습니다. 다만 같이 협업하는 동료가 이런 코드를 잘 이해할 수 있는 상황이어야 해요.